picard/ui/__init__.py

```diff
@@ -61,12 +61,38 @@ class PreserveGeometry:
 
     def __init__(self):
         Option("persist", self.opt_name(), QtCore.QByteArray())
+        Option("persist", self.splitters_name(), {})
         if getattr(self, 'finished', None):
             self.finished.connect(self.save_geometry)
 
     def opt_name(self):
         return 'geometry_' + self.__class__.__name__
 
+    def splitters_name(self):
+        return 'splitters_' + self.__class__.__name__
+
+    def _get_lineage(self, widget, lineage=''):
+        """Try to develop a unique lineage / ancestry string for the specified widget.
+
+        Args:
+            widget (QtWidget): Widget to process.
+            lineage (str, optional): Starting string. Defaults to ''. This is used during recursion and should not be specified in the initial call.
+
+        Returns:
+            str: Lineage / ancestry string for the specified widget.
+        """
+        if widget is not None:
+            # If widget doesn't have an assigned objectName() use the widget class name to help create a unique lineage string.
+            name = widget.objectName() if widget.objectName() else widget.__class__.__name__
```

Comment:
Since "names should be assigned if possible to help ensure that the lineage string key is unique for each splitter within the dialog", it would make sense to log (debug level) splitters not having an `objectName`.